TLM/TLM/Manager/Impl/OptionsManager.cs

```diff
@@ -264,24 +264,24 @@ private static void ToSlider(byte[] data, uint idx, ILegacySerializableOption op
                 save[40] = (byte)(Options.allowFarTurnOnRed ? 1 : 0);
                 save[41] = (byte)(Options.automaticallyAddTrafficLightsIfApplicable ? 1 : 0);
 
-                save[42] = OptionsMassEditTab.RoundAboutQuickFix_StayInLaneMainR.Save();
-                save[43] = OptionsMassEditTab.RoundAboutQuickFix_StayInLaneNearRabout.Save();
-                save[44] = OptionsMassEditTab.RoundAboutQuickFix_DedicatedExitLanes.Save();
-                save[45] = OptionsMassEditTab.RoundAboutQuickFix_NoCrossMainR.Save();
-                save[46] = OptionsMassEditTab.RoundAboutQuickFix_NoCrossYieldR.Save();
-                save[47] = OptionsMassEditTab.RoundAboutQuickFix_PrioritySigns.Save();
+                save[42] = PoliciesTab_RoundaboutsGroup.RoundAboutQuickFix_StayInLaneMainR.Save();
```

Comment:
Hmm, why it's not read from `Options.xyz` like the rest? Did I miss something (a comment)?

Reply:
This will be updated in future PR to make them all consistent one way or the other.

Ideally I'd like to make `Options.cs` fields instance based and have generic observable, etc., which will be a bunch more refactoring (expecting to do that for 11.6.5.2 branch).

Reply:
Ok, thanks.